ag_data/models.py

```diff
@@ -13,15 +22,22 @@ class AGEvent(models.Model):
     event_name = models.CharField(max_length=40, blank=True)
     event_date = models.DateTimeField(default=timezone.now)
     event_description = models.CharField(max_length=100, null=False, blank=True)
+    event_venue = models.ForeignKey(AGVenue, null=True, on_delete=models.SET_NULL)
+
+
+class AGSensorType(models.Model):
+
+    sensorType_id = models.AutoField(primary_key=True)
+    sensorType_name = models.CharField(max_length=1024, blank=True)
+    sensorType_processingFormula = models.IntegerField(default=0, null=False)
+    sensorType_format = JSONField()
```

Comment:
So it's a mixed style of camelCase and snake_case. In any way, sticking to one style and being consistent would be important.  Personally I prefer use snake_case for variable names when I work on python [as pep8 suggests](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles).

Reply:
Good point! Will use snake_case.

Reply:
My original idea is to use the underscore to separate the table name and the field name. But it is not used now.

Reply:
@alldne Good point, I am going to point that out on my on-call report